Sources/App/Core/SiteURL.swift

```diff
@@ -170,15 +181,15 @@ protocol Resourceable {
 
 extension Resourceable {
     func absoluteURL(anchor: String? = nil) -> String {
-        "\(Current.siteURL())/\(path)" + (anchor.map { "#\($0)" } ?? "")
+        "\(SiteURL.absoluteURL(path))" + (anchor.map { "#\($0)" } ?? "")
     }
 
     func absoluteURL(parameters: [String: String]) -> String {
-        "\(Current.siteURL())/\(path)\(parameters.queryString())"
+        "\(SiteURL.absoluteURL(path))\(parameters.queryString())"
     }
 
     func relativeURL(anchor: String? = nil) -> String {
-        "/" + path + (anchor.map { "#\($0)" } ?? "")
+        "\(SiteURL.relativeURL(path))" + (anchor.map { "#\($0)" } ?? "")
```

Comment:
This is just ensuring we route all url methods through the same two methods on `SiteURL`. Behaviour is unchanged (covered by `SiteURLTests`).